src/main/zlatopolsky/chapter10/Ch1N10_53.java

```diff
@@ -0,0 +1,14 @@
+package zlatopolsky.chapter10;
+
+import java.util.Scanner;
+
+public class Ch1N10_53 {
+
+    public void reverseRow(Scanner sc) {
```

Comment:
What this method does?
I tried to launch it, but it is just constantly asking for an input, and does nothing

Reply:
fix!: corrected name method, this method recursion reverse row while will not be zero